types/serve-static/serve-static-tests.ts

```diff
@@ -26,3 +29,22 @@ serveStatic.mime.define({
     'application/babylonmeshdata': ['babylonmeshdata'],
     'application/fx': ['fx']
 });
+
+serveStatic('/does-not-assume-express', {
+    setHeaders: function(res) {
+        // $ExpectError
+        res.set("foo", "bar");
+    }
+});
+
+http.createServer((req, res) => {
+    serveStatic('/works-with-node-server')(req, res, () => {});
+});
+
+app.use(serveStatic('/infers-express-response-when-passed-to-express-use', {
+    setHeaders: function(res) {
+        // $ExpectType Response<any, number>
```

Comment:
Not sure why but `express.Response` doesn't work, nor does `express.Response<any, number>` work. dtslint complains it needs to be `Response<any, number`>, so going with it.